src/frontends/pytorch/src/op/var_mean.cpp

```diff
@@ -76,6 +78,43 @@ OutputVector translate_var_mean(const NodeContext& context) {
     return {var, mean};
 };
 
+OutputVector translate_var_mean_fx(const NodeContext& context) {
+    num_inputs_check(context, 1, 4);
+    auto data = context.get_input(0);
+    bool unbiased = false;
+    auto num_elements = numel(context, data);
+    std::shared_ptr<ov::Node> mean, t_mean;
+    ov::Output<ov::Node> axes;
+
+    axes = context.get_input(1);
```

Comment:
the check of number of inputs does not look correct. Above you check that at least 1 input provided, but here you use the second input without any checks that context contains it